htlcswitch/interceptable_switch.go

```diff
@@ -416,6 +445,60 @@ func (s *InterceptableSwitch) resolve(res *FwdResolution) error {
 	}
 }
 
+func (s *InterceptableSwitch) validateChannelUpdate(
+	update *lnwire.ChannelUpdate) error {
+
+	// The maxHTLC flag is mandatory.
+	if !update.MessageFlags.HasMaxHtlc() {
```

Comment:
More validation is needed here, eg: max HTLC shouldn't be greater than the channel capacity, etc, etc. I think we should just call the normal validation routine.